stories/core/ComboBox.stories.tsx

```diff
@@ -365,3 +365,25 @@ export const DisabledItems: Story<Partial<ComboBoxProps<string>>> = (args) => {
   );
 };
 DisabledItems.args = { ...Basic.args };
+
+export const WithLabel: Story<Partial<ComboBoxProps<string>>> = (args) => {
+  const options = React.useMemo(() => countriesList, []);
+
+  return (
+    <>
+      <Label htmlFor='combo-input'>Country</Label>
```

Comment:
I am not sure if I should add a separate story for this. There is no use for a label on its own.

Reply:
Maybe add story for `Label` in Typography section. Have examples with combobox and other inputs

Reply:
I agree with Greta. If we don't have a dedicated story then people might not even know about this component.

Reply:
Added stories under Typography